x-pack/plugins/ml/public/application/datavisualizer/file_based/components/results_links/results_links.tsx

```diff
@@ -72,9 +68,19 @@ export const ResultsLinks: FC<Props> = ({
       if (globalState?.time) {
         state.timeRange = globalState.time;
       }
+
+      let discoverUrlGenerator;
+      try {
+        discoverUrlGenerator = getUrlGenerator(DISCOVER_APP_URL_GENERATOR);
```

Comment:
getUrlGenerator throws when the id is not found. Adding code to avoid unhandled errors